assignments/objects.js

```diff
@@ -30,7 +30,8 @@ const kennan = {
   "id": 2,
   "name": "Kennan",
   "email": "kdiben1@tinypic.com",
-  "gender": "M"
+  "gender": "M",
+  speak: function(){return "Hello, my name is Kennan!";}
```

Comment:
you could use this key word for the speak method. in this context `this` refers to it's self. an example of this would like this

```return `Hello, my name is ${this.name}!` ```